charts/external-dns/crds/dnsendpoint.yaml

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
```suggestion
```
This is a single YAML document and _Helm_ will add document separators where necessary.